docs/index.rst

```diff
@@ -0,0 +1,191 @@
+   * The processed BAM file is available `here <https://portal.nersc.gov/cfs/m3408/test_data/metaT/SRR11678315/assembly_output/SRR11678315-int-0.1_pairedMapped_sorted.bam>`_
+   * The proccessed GFF functional annotation file is available `here <https://portal.nersc.gov/cfs/m3408/test_data/metaT/SRR11678315/annotation_output/SRR11678315-int-0.1_functional_annotation.gff>`_
+   * The generated mapping file file is available `here <https://portal.nersc.gov/cfs/m3408/test_data/metaT/SRR11678315/annotation_output/SRR11678315-int-0.1_contig_names_mapping.tsv>`_
+   * The sample read count outputs are available `here <https://portal.nersc.gov/cfs/m3408/test_data/metaT/SRR11678315/readcounts_output/>`_
+
+
+
+Input
+------
+
+A JSON file containing the following: 
+
+#. project name 
+#. BAM File 
+#. GFF Functional Annotation File
+#. (optional) Map file
+#. (optional) RNA type
+
+An example JSON file is shown below:
+
+.. code-block:: JSON
+   {
+      "readcount.proj_id":"SRR11678315-int-0.1",
+      "readcount.bam": "https://portal.nersc.gov/cfs/m3408/test_data/metaT/SRR11678315/assembly_output/SRR11678315-int-0.1_pairedMapped_sorted.bam",
+      "readcount.gff": "https://portal.nersc.gov/cfs/m3408/test_data/metaT/SRR11678315/annotation_output/SRR11678315-int-0.1_functional_annotation.gff",
+      "readcount.map": "https://portal.nersc.gov/cfs/m3408/test_data/metaT/SRR11678315/annotation_output/SRR11678315-int-0.1_contig_names_mapping.tsv"
+   }
+
+
+The map file connects the naming schemes between the GFF and BAM files. If the naming scheme is the same, the map file can either be generated automatically if none is specified, or user can make a tsv with two columns of the names from the GFF file. 
+The RNA type inputs are include nothing, :code:`aRNA`, or :code:`non_stranded_RNA`, which are transformed to script inputs :code:`(default blank)`, :code:`-aRNA yes`, or :code:`-non_stranded yes`, respectively. This is the explanation from the script itself:
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+   * - :code:`aRNA yes`
+     - use antisense reads during counting, default no
+   * - :code:`non_stranded yes`
+     - for cDNA input, default no, override "-aRNA yes" if "-non_stranded yes"
+
+
+
+
+Output
+------
+
+The output will have one directory named by prefix project name and output files including count statistics, status log, and run information. 
+
+The main read count table output is named by prefix.readcount. 
+ 
+.. list-table:: 
+   :header-rows: 1
+
+   * - File Name
+     - Description
+   * - nmdc_xxxxxxx.rnaseq_gea.txt
+     - main read count table output
+   * - nmdc_xxxxxxx.rnaseq_gea.intergenic.txt
```

Comment:
Consider dropping the intergenic file. This was only happening when the input file was from the IMG database instead of Marcel's file in jamo. Both JGI and NMDC now use the output of Marcel's workflow as the input to this workflow.